Owncloud iOs Client/Tabs/SettingTab/SettingsViewController.m

```diff
@@ -1762,260 +1769,21 @@ -(void)setPropertiesTouchIDToState:(BOOL)isTocuhIDActive{
     }
 }
 
-
-#pragma mark - Instant Upload - Location Support
-
-- (void)switchInstantUploadTo:(BOOL)value {
-     [self.switchInstantUpload setOn:value animated:NO];
-}
-
--(void)setPropertiesInstantUploadToState:(BOOL)stateInstantUpload{
-    
-    AppDelegate *app = (AppDelegate *)[[UIApplication sharedApplication] delegate];
-    if (app.activeUser.username == nil) {
-        app.activeUser = [ManageUsersDB getActiveUser];
-    }
-    
-    if (stateInstantUpload) {
-        [self switchInstantUploadTo:YES];
-        app.activeUser.instantUpload = YES;
-        [ManageAppSettingsDB updateInstantUploadTo:YES];
-    } else {
-        [self switchInstantUploadTo:NO];
-        app.activeUser.instantUpload = NO;
-        [ManageAppSettingsDB updateInstantUploadTo:NO];
-        [[ManageLocation sharedSingleton] stopSignificantChangeUpdates];
-    }
-}
-
--(void)setDateInstantUpload{
```

Comment:
We used this to activate the instant upload using the current date. The idea it is not upload the old pictures... just start with the new ones.
Because you remove that code the Instant Uploads will upload all the images that are on the device